components/brave_ads/core/internal/common/net/http/http_status_code.h

```diff
@@ -8,8 +8,9 @@
 
 namespace net {
 
+// HTTP status code 418 "I'm a teapot" is an April Fools' joke from the IETF's
+// RFC 2324, Hyper Text Coffee Pot Control Protocol (HTCPCP/1.0).
```

Comment:
```suggestion
// RFC 2324, Hyper Text Coffee Pot Control Protocol (HTCPCP/1.0).
// We only use it in tests, it's not used in production.
```

Reply:
Unfortunately this is used in production for confirmations (non-Rewards).